cmd/edit/cluster/cmd.go

```diff
@@ -463,6 +541,91 @@ func run(cmd *cobra.Command, _ []string) {
 		}
 	}
 
+	/*******  Registries Config *******/
+	updateRegistriesConfig := false
+	if insecureRegistries != nil || blockedRegistries != nil || allowedRegistriesForImport != nil ||
+		allowedRegistries != nil || platformAllowlist != "" || additionalTrustedCa != "" {
+		updateRegistriesConfig = true
+	}
+
+	if !updateRegistriesConfig && interactive.Enabled() {
+		updateRegistriesConfigValue, err := interactive.GetBool(interactive.Input{
+			Question: "Update registries config",
+			Default:  updateRegistriesConfig,
+		})
+		if err != nil {
+			r.Reporter.Errorf("Expected a valid value: %s", err)
+			os.Exit(1)
+		}
+		updateRegistriesConfig = updateRegistriesConfigValue
+	}
+
+	if updateRegistriesConfig && interactive.Enabled() {
+		allowedRegistriesInputs, err := interactive.GetString(interactive.Input{
+			Question: "Allowed Registries",
+			Help:     cmd.Flags().Lookup("allowed-registries").Usage,
+			Default:  allowedRegistries,
+		})
+		if err != nil {
+			r.Reporter.Errorf("Expected a valid value for allowed registries: %s", err)
+			os.Exit(1)
+		}
+		allowedRegistries = helper.HandleEmptyStringOnSlice(strings.Split(allowedRegistriesInputs, ","))
+
+		blockedRegistriesInputs, err := interactive.GetString(interactive.Input{
+			Question: "Blocked Registries",
+			Help:     cmd.Flags().Lookup("blocked-registries").Usage,
+			Default:  blockedRegistries,
+		})
+		if err != nil {
+			r.Reporter.Errorf("Expected a valid value for blocked registries: %s", err)
+			os.Exit(1)
+		}
+		blockedRegistries = helper.HandleEmptyStringOnSlice(strings.Split(blockedRegistriesInputs, ","))
+
+		insecureRegistriesInputs, err := interactive.GetString(interactive.Input{
+			Question: "Insecure Registries",
+			Help:     cmd.Flags().Lookup("insecure-registries").Usage,
+			Default:  insecureRegistries,
+		})
+		if err != nil {
+			r.Reporter.Errorf("Expected a valid value for insecure registries: %s", err)
+			os.Exit(1)
+		}
+		insecureRegistries = helper.HandleEmptyStringOnSlice(strings.Split(insecureRegistriesInputs, ","))
+
+		allowedRegistriesForImportInputs, err := interactive.GetString(interactive.Input{
+			Question: "Allowed Registries For Import",
+			Help:     cmd.Flags().Lookup("allowed-registries-for-import").Usage,
+			Default:  allowedRegistriesForImport,
+		})
+		if err != nil {
+			r.Reporter.Errorf("Expected a valid value for allowed registries for import: %s", err)
+			os.Exit(1)
+		}
+		allowedRegistriesForImport = helper.HandleEmptyStringOnSlice(strings.Split(allowedRegistriesForImportInputs, ","))
+
+		platformAllowlist, err = interactive.GetString(interactive.Input{
+			Question: "Platform Allowlist",
+			Help:     cmd.Flags().Lookup("platform-allowlist").Usage,
+			Default:  platformAllowlist,
+		})
+		if err != nil {
+			r.Reporter.Errorf("Expected a valid value for platform allowlist: %s", err)
+			os.Exit(1)
+		}
+
+		additionalTrustedCa, err = interactive.GetString(interactive.Input{
+			Question: "Additional Trusted CA file path",
```

Comment:
It seems to me that a path to a file containing a single certificate is required here, not a certificate chain or a keystore, correct?

In both cases, I think we should specify whether we expect the input file to be DER or PEM encoded.

Reply:
It actually expects it in a json in {registry.io: cert content} format, let me add a validation for it.

Reply:
Added the validator